test/4.1.12.sh

```diff
@@ -5,5 +5,5 @@
 
 priv_cmds="$(find / -xdev \( -perm -4000 -o -perm -2000 \) -type f)"
 for cmd in ${priv_cmds} ; do
-        cut -d\# -f1 /etc/audit/audit.rules | egrep "\-k[[:space:]]+privileged" | egrep "\-F[[:space:]]+path=${cmd}" | egrep "\-F[[:space:]]+perm=x" | egrep "\-F[[:space:]]+auid>=500" | egrep "\-F[[:space:]]+auid\!=4294967295" | egrep -q "\-a[[:space:]]+always,exit|\-a[[:space:]]+exit,always" || exit 1
+        cut -d\# -f1 /etc/audit/audit.rules | egrep "\-k[[:space:]]+privileged" | egrep "\-F[[:space:]]+path=${cmd}" | egrep "\-F[[:space:]]+perm=x" | egrep "\-F[[:space:]]+auid>=1000" | egrep "\-F[[:space:]]+auid\!=4294967295" | egrep -q "\-a[[:space:]]+always,exit|\-a[[:space:]]+exit,always" || exit 1
```

Comment:
Did you also have to check inside `/etc/audit/rules.d/*`?

Reply:
No, I only made changes to the /etc/audit/audit.rules file so the rules.d file is unaltered.  Additionally I did not find of the tests that I modified that any other looked in that file, only the /etc/audit/audit.rules file.

Reply:
Do we have rules in /etc/audit/rules.d that may contain the things this is checking for though? Only the rules.d file is installed when we make the CentOS image.

Reply:
If that is the case I can always copy over the changes I made in the /etc/audit/audit.rules since I only added information.  That way we can be on the safe side and make sure both files have the same information.